scripts/link-checker/check-links.js

```diff
@@ -306,6 +306,9 @@ function excludeAcceptable(links) {
         // intelligently, but we can come back to that in a follow up.
         .filter(b => !(b.reason === "HTTP_429" && b.destination.match(/github.com|npmjs.com/)))
 
+        // Ignore GitHub 403s
+        .filter(b => !(b.reason === "HTTP_403" && b.destination.match(/github.com/)))
```

Comment:
maybe we should ignore all 403s not just github ones 🤷🏻

Reply:
Before we just go ignoring, do we have some idea why these are returning 403 to begin with?

Reply:
i thought you wrote why in slack...

Reply:
https://pulumi.slack.com/archives/C03G209S6H4/p1654010275582649

Reply:
if you have another idea pls lets do that, the noise of the 403s is preventing us from seeing and fixing the 404s.